src/boot/efi/util.h

```diff
@@ -129,14 +129,16 @@ static inline void unload_imagep(EFI_HANDLE *image) {
                 (void) BS->UnloadImage(*image);
 }
 
+#define MAKE_GUID(GUID) ((EFI_GUID) GUID##_GUID)
```

Comment:
so if we decide to pass around guids as in userspace, then that#s cetrainly something we can discuss. But if so, then let's try to make the macro more careful, i.e. let's keep the const in to give the compiler the hint that the literal data actually *is* constant, but then cast the const away to work around UEFI's stupid api definitoins.

i.e. maybe like so:

```c
#define MAKE_CONST_GUID(g) ((const EFI_GUID) g##_GUID)
#define MAKE_GUID(g) (*(EFI_GIUD*) &MAKE_CONST_GUID(g))
```

untested, but I think this should work.